C-T-M.cpp

```diff
@@ -124,45 +140,48 @@ void retrieve_details(char index_no[], char file_name[])
 {
 
     int no = get_num_records(file_name);
+    cout<<no<<" numbers";
 
     for (int i = 0; i < no; i++)
     {
-
         if (strcmp(stdrec[i].ind, index_no) == 0)
         {
             cout << "\n\n"
-                 << "Details : ";
+                 << "Details :\n";
             write_details(file_name);
-            cout << "\t" << stdrec[i].field1 << "\t" << stdrec[i].field2 << "\t" << stdrec[i].field3 << "\t" << stdrec[i].field4 << "\t" << stdrec[i].field5 << "\n";
+            cout << "\t" << stdrec[i].field1 << "\t\t" << stdrec[i].field2 << "\t\t" << stdrec[i].field3 << "\t\t" << stdrec[i].field4 << "\t\t" << stdrec[i].field5 << "\n";
             break;
         }
     }
 }
 
-void search_record(string filename)
+void search_record(string filename,string indFilename)

```

Comment:
![Screenshot (491)](https://user-images.githubusercontent.com/59915937/128197566-295ef70b-db9f-4905-95df-0094047814da.png)

some issue,check again if its working fine in yours